UPGRADE-NOTES.md

```diff
@@ -0,0 +1,40 @@
+# Upgrade notes
+
+When you upgrade your Mattermost containers, please check at those upgrade notes for some breaking changes.
+
+## Upgrading Mattermost to 6.0+
+
+Starting with Mattermost `6.0` :
+- we stopped to support the `web` Docker image (you can find more information about it on [#245](https://github.com/mattermost/mattermost-docker/issues/245))
+- we stopped to maintain a "too generic" `docker-compose.yml` file
+
+If you were not using the `web` image and the `docker-compose.yml` file provided in this repository, nothing changed for you.  
+
+If you used the `web` image or the `docker-compose.yml` file, we invite you to setup your own reverse proxy and to write your own Docker Compose file. A good starting point is to look at the [Traefik deployment example](contrib/traefik/README.md) that is close to the previous Docker Compose file and support TLS. **Please backup your existing volumes, data and configuration** before the migration, in this example the path to Docker bind mounts changed. Ensure you move all your data (with correct access rights) to the right folders.
```

Comment:
We can also point people at our official nginx setup instructions to pull the config from there.

Reply:
Good point, we can suggest to use the [Nginx docker image](https://hub.docker.com/_/nginx) with [this configuration](https://docs.mattermost.com/install/install-debian-88.html#configuring-nginx-as-a-proxy-for-mattermost-server) (which is basically the `web` image).

Reply:
I tried to improve upgrade notes and readme with this information.